source/Nevermore/IQueryBuilder.cs

```diff
@@ -213,6 +213,13 @@ public interface IQueryBuilder<TRecord> where TRecord : class
         /// <returns>The query builder that can be used to further modify the query, or execute the query</returns>
         IQueryBuilder<TRecord> AddRowNumberColumn(string columnAlias, params ColumnFromTable[] partitionByColumns);
 
+        /// <summary>
```

Comment:
Interesting. Personally I think we shouldn't do this, but I'm not confident in that decision :)

Just because two methods look the same, doesn't mean they are the same. To me, the `IParameterNameGenerator` felt more like an implementation detail of nevermore (arguably it shouldn't be public then, but you could argue the same thing about most of the classes in this assembly that are currently public). While the method on `IQueryBuilder` was the public API for generating parameter names. Therefore, they might diverge in future.